daemon/protocol/python/scoot/daemon_cli.py

```diff
@@ -0,0 +1,156 @@
+"""
+Usage:
+  daemon_cli.py snapshot create <srcDir>
+  daemon_cli.py snapshot checkout <snapshotId> <destDir>
+  daemon_cli.py exec run <command> [--snapshotId=<sid>] [--timeout=<ns>]
+  daemon_cli.py exec poll <runIds> [--wait=<wait>] [--all]
```

Comment:
perhaps we should have a quick huddle to review the current command line and discuss what we want as final state.